runtime/lib/apis/mqtt-registry.js

```diff
@@ -6,7 +6,8 @@ const crypto = require('crypto');
 const context = require('@rokid/context');
 
 // TODO(Yorkie): move something where?
-const host = 'mqtt-dev-registry.rokid.com';
+// const host = 'mqtt-dev-registry.rokid.com';
+const host = 'wormhole-registry.rokid.com';
```

Comment:
use https instead of http

Reply:
I have used https at https://github.com/Rokid/rokidos-node/blob/beb9637d62045e7533411e189adeab4ffb9eab5b/runtime/lib/apis/mqtt-registry.js#L34 :)